standards/accessibility.md

```diff
@@ -0,0 +1,30 @@
+# Accessibility Standards
+
+The Library provides resources that should be accessible to all patrons. When building front-end applications that patrons interact with, we MUST build the apps with accessibility in mind. This is a general overview and the Design Toolkit's [Developer Checklist](https://nypl.github.io/design-toolkit/resources/development-checklist.html) MUST be followed.
```

Comment:
I would promote saying we MUST build all applications in an accessibility first approach. Ensuring the development process integrates all a11y standards from the beginning. 🍔